database_connector.py

```diff
@@ -0,0 +1,33 @@
+import pymysql.cursors
+
+
+class DatabaseConnector:
+    def __init__(self, database=None):
+        self.host = 'localhost'
+        self.user = 'root'
+        self.password = ''
+        self.db = database
+        self.charset = 'utf8mb4'
```

Comment:
If these wont change they can be class variables:
https://docs.python.org/2/tutorial/classes.html#class-and-instance-variables